### PR TITLE
Bitstamp: Fix USDT balance not loading

### PR DESCRIPTION
**Pull request by gbjk**
## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run
- [x] Test TestGetBalances

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shazbert — APPROVED** [review]
tACK.

File: exchanges/bitstamp/bitstamp.go [hunk 1]
```diff
@@ -228,7 +228,10 @@ func (b *Bitstamp) GetBalance(ctx context.Context) (Balances, error) {
 	}
 	balances := make(map[string]Balance)
 	for k := range balance {
-		curr := k[0:3]
+		curr, _, found := strings.Cut(k, "_")
```
Comment (gloriousCode) [inline exchanges/bitstamp/bitstamp.go:231]: It works specifically for what you're after, but it is incompatible with the fee section below.
You're splitting on the `_`, but then the code from lines 255-259 rely on appending a currency after it then splitting on the underscore. 

With no error checking on anything below (which is not on you), its hard to spot, but we end up with looking for a map entry of`zrxeurbtc_fee` when we're looking for `zrxeur_fee` to apply to the `eurFree` field. So this introduces some extra bugs

On this branch we get the following response:
```
          "BTC": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0.0005,
          "BTCFee": 0,
          "USDFee": 0,
          "EURFee": 0
         },
         "BTCEUR": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0,
          "BTCFee": 0,
          "USDFee": 0,
          "EURFee": 0
         },
```

on `master` we get:

```
  "BTC": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0,
          "BTCFee": 0,
          "USDFee": 0.25,
          "EURFee": 0.25
         },
```

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thanks for finding this bug. Unfortunately it has some extra problems. Happy to work with you through them

File: exchanges/bitstamp/bitstamp.go (hunk 1 above)
Comment (gbjk) [reply exchanges/bitstamp/bitstamp.go:231]: No problem. Will fix.

Comment (gbjk) [reply exchanges/bitstamp/bitstamp.go:231]: Just for fun: These values are 0 in live api always 😂

Comment (gbjk) [reply exchanges/bitstamp/bitstamp.go]: Okay. So I've fixed this, but also uncovered that I'm pretty sure bitstamp pulled the rug on the v1 schema and returns 0 for all old *_fee fields.
It now just uses withdrawalFee.

**Comment by gbjk** [comment]
@gloriousCode Fixes pushed. Ready for re-review

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks for the changes. I'm hoping their next API is nicer than this 😄

**Review by thrasher- — APPROVED** [review]
Thanks for expanding the test coverage as well 😎

**Pull request merged**